docs/guide/media.md

```diff
@@ -0,0 +1,167 @@
+# Working with images and media files
+
+There are a few ways to configure a field schema to allow editors to select images or other media files.
+
+1. Attachment fields
+2. Relationship fields to the image or file piece types
+3. Area fields with the image widget
```

Comment:
I would recommend making the wording of this list match both the navigation on the sidebar, and the headers in the content as well. So you can clearly draw the connection between all three ways to find the chunk of content most applicable to your need. I would also recommend making the ordering of the list match the ordering of the navigation and the sections.